lib/vmpooler/pool_manager.rb

```diff
@@ -50,7 +50,10 @@ def check_pending_vm(vm, pool, timeout)
             rescue
             end
 
+            finish = '%.2f' % (Time.now - Time.parse($redis.hget('vmpooler__vm__' + vm, 'clone')))
```

Comment:
Please add a guard for the `redis.hget`. If that comes back without "time information", `Time.parse` will explode.